test/setupTests.js

```diff
@@ -12,6 +12,10 @@ const deleteTables = () => {
   return prisma.$transaction(deleteTables)
 }
 
+// added to adress the tests issue where if you start with a non empty DB the first time you running the tests the first test (GET /movies) fails.
+global.beforeAll(() => {
+  return deleteTables()
+})
```

Comment:
Love it! If you do a new fork of the cinema repo, add this in and create a PR with just this (and no comment above it), I'll merge this into the exercise. You'll be the first student to contribute to a primary exercise repo :)